vowpalwabbit/global_data.h

```diff
@@ -537,7 +532,9 @@ struct vw
   time_t init_time;
 
   std::string final_regressor_name;
-  regressor reg;
+
+  uint32_t stride_shift;
```

Comment:
This loss of modularity of stride_shift is annoying.  I'd like to hide this entirely inside of the parameters, but that requires more work than seems reasonable at this point.